src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +55,14 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/**
+ * Modify relevant parts of the rendered DOM tree under the given root,
+ * inclusive, to ensure functionality.
+ *
+ * If no root element is specified, the entire document is considered.
```

Comment:
This feature had been marked as "DEPRECATED". Is there a reason to drop that?

Reply:
I did not know the correct meaning of deprecated. I thought it meant 'no longer used' when I removed it - because it was being used and so I thought that the "DEPRECATED" tag was wrong.

Just now I searched for the meaning, and it means "usable but regarded as obsolete and best avoided, typically because it has been superseded", so the comment is not incorrect. I'll revert it.